src/components/footer/Footer.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <footer>
+        <p class="footerText">
+            Made by <a href="https://github.com/Androz2091">Androz2091</a> in France
+        </p>
+        <div class="footerIcons">
+            <a href="https://androz2091.fr/discord">
+                <span class="fa-stack fa-1x">
+                    <i class="socialIconBack fas fa-circle fa-stack-2x"></i>
+                    <i class="socialIconTop fab fa-discord fa-stack-1x"></i>
+                </span>
+            </a>
+            <a href="https://github.com/dumpus-app">
+                <span class="fa-stack fa-1x">
+                    <i class="socialIconBack fas fa-circle fa-stack-2x"></i>
+                    <i class="socialIconTop fab fa-github fa-stack-1x"></i>
+                </span>
+            </a>
+            <a href="mailto:contact@dumpus.app">
+                <span class="fa-stack fa-1x">
+                    <i class="socialIconBack fas fa-circle fa-stack-2x"></i>
+                    <i class="socialIconTop fas fa-envelope fa-stack-1x"></i>
+                </span>
+            </a>
+        </div>
+        <div class="footerLinks">
+            <a href="/" target="_self">Home</a>
```

Comment:
```suggestion
            <router-link to="/">Home</router-link>
```
isn't it better to use router link to prevent reloads?

Reply:
have fixed it